turbinia/task_manager.py

```diff
@@ -95,8 +97,15 @@ def task_runner(obj, *args, **kwargs):
   Returns:
     Output from TurbiniaTask (should be TurbiniaTaskResult).
   """
-  obj = workers.TurbiniaTask.deserialize(obj)
-  return obj.run_wrapper(*args, **kwargs)
+  if os.path.exists(config.LOCK_FILE) or os.path.exists(config.TO_BE_DELETED):
+    raise psq.Retry()
+  else:
+    with filelock.FileLock(config.LOCK_FILE):
+      obj = workers.TurbiniaTask.deserialize(obj)
+      run = obj.run_wrapper(*args, **kwargs)
+    if os.path.exists(config.LOCK_FILE):
```

Comment:
Was there an issue that required this?  As far as I know, this should never happen, so I just wanted to check that there wasn't some other issue surfacing here (e.g. filelock not handling exceptions properly, etc).

Reply:
Per our chat updated this whole function to try except statement and catch psq.Retry()